watertap/unit_models/electroNP_ZO.py

```diff
@@ -339,107 +175,16 @@ def MgCl2_demand(b, t):
-
-        Returns:
-            None
-        """
-        if optarg is None:
-            optarg = {}
-
-        # Set solver options
-        init_log = idaeslog.getInitLogger(self.name, outlvl, tag="unit")
-        solve_log = idaeslog.getSolveLogger(self.name, outlvl, tag="unit")
-
-        solver_obj = get_solver(solver, optarg)
-
-        # Get initial guesses for inlet if none provided
-        if state_args is None:
-            state_args = {}
-            state_dict = self.properties_in[
-                self.flowsheet().time.first()
-            ].define_port_members()
-
-            for k in state_dict.keys():
-                if state_dict[k].is_indexed():
-                    state_args[k] = {}
-                    for m in state_dict[k].keys():
-                        state_args[k][m] = state_dict[k][m].value
-                else:
-                    state_args[k] = state_dict[k].value
-
-        # ---------------------------------------------------------------------
-        # Initialize control volume block
-        flags = self.properties_in.initialize(
-            outlvl=outlvl,
-            optarg=optarg,
-            solver=solver,
-            state_args=state_args,
-            hold_state=True,
-        )
-        self.properties_treated.initialize(
-            outlvl=outlvl,
-            optarg=optarg,
-            solver=solver,
-            state_args=state_args,
-            hold_state=False,
-        )
-        self.properties_byproduct.initialize(
-            outlvl=outlvl,
-            optarg=optarg,
-            solver=solver,
-            state_args=state_args,
-            hold_state=False,
-        )
-
-        init_log.info_high("Initialization Step 1 Complete.")
-
-        # ---------------------------------------------------------------------
-        # Solve unit
-        with idaeslog.solver_log(solve_log, idaeslog.DEBUG) as slc:
-            results = solver_obj.solve(self, tee=slc.tee)
-
-        init_log.info_high(
-            "Initialization Step 2 {}.".format(idaeslog.condition(results))
-        )
-
-        # ---------------------------------------------------------------------
-        # Release Inlet state
-        self.properties_in.release_state(flags, outlvl)
-
-        init_log.info("Initialization Complete: {}".format(idaeslog.condition(results)))
-
-        if not check_optimal_termination(results):
-            raise InitializationError(
-                f"{self.name} failed to initialize successfully. Please check "
-                f"the output logs for more information."
-            )
-
     def _get_performance_contents(self, time_point=0):
         var_dict = {}
         var_dict["Water Recovery"] = self.recovery_frac_mass_H2O[time_point]
         for j in self.config.property_package.solute_set:
-            var_dict[f"Solute Removal {j}"] = self.removal_frac_mass_comp[time_point, j]
+            var_dict[f"Solute Removal {j}"] = self.split_fraction[0, "byproduct", j]
```

Comment:
I just want to point out that `removal_frac_mass_comp` was a variable that was created for all the zero order models that we worked on previously (for WaterTAP3 migration as well as 1st year IEDO 2336 effort). 

Although we made this unit model to deviate away from the zero-order model framework, I do wonder if it would be worthwhile to preserve the same naming conventions (i.e., through object references so we don't lose the new names, such as split fraction). If we ever want to make use of an array of zero-order models down the line, say, to do that superstructure optimization with BSM2 and a set of zero-order model options, including this one, I could see variations in naming being an obstacle to a more streamlined analysis effort. 

Not that serious, but something to be aware of.